README.md

```diff
@@ -1 +1,151 @@
+  <h3><b>Haiti Tech Summit 2023</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [👥 Aknowledgements](#aknowledgements)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+
+# 📖 Haiti Tech Summit <a name="about-project"></a>
+
+
+**Haiti Tech Summit 2023** is a technology event that takes place every year in Haiti on a different theme, with speakers from around the world presenting their topics around the theme of the event.
+This year, the theme chosen for HTS 2023 is "Artificial Intelligence, are we talking?"
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML/CSS</summary>
+  <summary>Javascript</summary>
+  <summary>Linters</summary>
+  <summary>Github</summary>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Adding pictures**
+- **Using flex**
+- **Side bar menu**
+- **Responsive app**
+- **Adding js script**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Click on this link to visit the project : https://menendezon.github.io/HaitTechSubmit/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps. It's a simple static project.
+
+### Prerequisites
+
+In order to run this project, you need :
+- Git to clone it (optional) 
+- A browser to run it
+- Some skill in HTML/CSS and js
+- An IDE
+- Install linters (optional)
+
+### Setup
+
+This project doesn't need something to install, run this command to download the project : 
+git clone git@github.com:MenendezON/HaitTechSubmit.git
+
+### Install
+
+To install the project, run this command :
+**npm install**
+
+### Usage
+
+To run the project on your computer, execute the following command :
+**npm start**
+
+### Run tests
+
+To run the project, execute the following command :
+**npm test**
+
+### Deployment
+
+This project has been deployed at this address https://menendezon.github.io/HaitTechSubmit/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Software developer**
+
+- GitHub: [@menendezon](https://github.com/menendezon)
+- Twitter: [@menendezon](https://twitter.com/menendezon)
+- LinkedIn: [MenendezON](https://linkedin.com/in/menendezon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Aknowledgements <a name="aknowledgements"></a>
```

Comment:
- [OPTIONAL] You did great adding this section as suggested by the previous reviewer and also adding the appropriate content. :+1: :100:  but there seems to be a spelling error here. I believe it should be spelled `Acknowledgements`. I'll suggest you correct this as a spelling error-free readme makes your report appear more professional. :smile: :small_airplane: